app/src/main/res/layout/activity_products_page.xml

```diff
@@ -21,10 +21,13 @@
         android:layout_marginRight="45dp"
         android:background="@drawable/ic_search_bar"
         android:ems="12"
-        android:hint="Search"
+        app:queryHint="Search"
```

Comment:
let's move this to strings.xml in case we'd like to change it  easily. I'm pretty sure there is already a string there for Search (maybe I'm wrong)

Reply:
That was the plan, I just wanted you guys to be able to workk with the new design

Reply:
Done